### PR TITLE
Properly handle Totem Pokemon

### PR DESCRIPTION
**Pull request by KrisOnline**

### CODE REVIEW

**Comment by Zarel** [comment]
Is this necessary? Totem Pokémon don't have stats differences

**Comment by AustinXII** [comment]
This isn’t necessary at all and I already handled this in the move set importer

**Comment by Marty-D** [comment]
They have weight differences, which is definitely not something that is currently handled.

**Comment by AustinXII** [comment]
Yeah someone just pointed that out on smogon, I guess this is what it was trying to fix, anyways yeah you’re going to have to remove the instances in ``moveset_import.js``

**Comment by KrisOnline** [comment]
ok done @AustinXII

**Comment by AustinXII** [comment]
Rad ty

File: js/data/pokedex.js [hunk 1]
```diff
@@ -11647,7 +11663,21 @@ var POKEDEX_SM = $.extend(true, {}, POKEDEX_XY, {
...
+		"t1": "Normal",
+		"bs": {
+			"hp": 88,
+			"at": 110,
+			"df": 60,
+			"sa": 55,
+			"sd": 60,
+			"sp": 45
+		},
+		"w": 60,
```
Comment (AustinXII) [inline js/data/pokedex.js]: The standard is ``60.0`` :/

File: js/data/pokedex.js [hunk 2]
```diff
@@ -11775,6 +11821,21 @@ var POKEDEX_SM = $.extend(true, {}, POKEDEX_XY, {
...
+		"t1": "Grass",
+		"bs": {
+			"hp": 70,
+			"at": 105,
+			"df": 90,
+			"sa": 80,
+			"sd": 90,
+			"sp": 45
+		},
+		"w": 58,
```
Comment (AustinXII) [inline js/data/pokedex.js]: Etc

File: js/data/pokedex.js [hunk 3]
```diff
@@ -11854,6 +11916,21 @@ var POKEDEX_SM = $.extend(true, {}, POKEDEX_XY, {
...
+		"t2": "Ghost",
+		"bs": {
+			"hp": 60,
+			"at": 80,
+			"df": 110,
+			"sa": 50,
+			"sd": 80,
+			"sp": 45
+		},
+		"w": 98,
```
Comment (AustinXII) [inline js/data/pokedex.js]: Etc

File: js/data/pokedex.js [hunk 4]
```diff
@@ -12294,6 +12394,21 @@ var POKEDEX_SM = $.extend(true, {}, POKEDEX_XY, {
...
+		"t2": "Normal",
+		"bs": {
+			"hp": 75,
+			"at": 71,
+			"df": 70,
+			"sa": 40,
+			"sd": 80,
+			"sp": 77
+		},
+		"w": 105,
```
Comment (AustinXII) [inline js/data/pokedex.js]: Etc

File: js/data/pokedex.js [hunk 5]
```diff
@@ -12319,7 +12434,23 @@ var POKEDEX_SM = $.extend(true, {}, POKEDEX_XY, {
...
+		"t2": "Fairy",
+		"bs": {
+			"hp": 60,
+			"at": 55,
+			"df": 60,
+			"sa": 95,
+			"sd": 70,
+			"sp": 124
+		},
+		"w": 2,
```
Comment (AustinXII) [inline js/data/pokedex.js]: Etc

File: js/data/pokedex.js [hunk 6]
```diff
@@ -12371,6 +12502,22 @@ var POKEDEX_SM = $.extend(true, {}, POKEDEX_XY, {
...
+		"t2": "Fire",
+		"bs": {
+			"hp": 68,
+			"at": 64,
+			"df": 60,
+			"sa": 111,
+			"sd": 60,
+			"sp": 117
+		},
+		"w": 81,
```
Comment (AustinXII) [inline js/data/pokedex.js]: And so on

Comment (KrisOnline) [reply js/data/pokedex.js]: I am SORRY OK

**Pull request merged**